audio.go

```diff
@@ -0,0 +1,67 @@
+package openrtb
+
+import (
+	"encoding/json"
+	"errors"
+)
+
+// Validation errors
+var (
+	ErrInvalidAudioNoMimes = errors.New("openrtb: audio has no mimes")
+)
+
+// The "audio" object must be included directly in the impression object
+type Audio struct {
+	Mimes         []string  `json:"mimes,omitempty"`       // Content MIME types supported.
+	MinDuration   int       `json:"minduration,omitempty"` // Minimum video ad duration in seconds
+	MaxDuration   int       `json:"maxduration,omitempty"` // Maximum video ad duration in seconds
+	Protocols     []int     `json:"protocols,omitempty"`   // Video bid response protocols
+	StartDelay    int       `json:"startdelay,omitempty"`  // Indicates the start delay in seconds
+	Sequence      int       `json:"sequence,omitempty"`    // Default: 1
+	BAttr         []int     `json:"battr,omitempty"`       // Blocked creative attributes
+	MaxExtended   int       `json:"maxextended,omitempty"` // Maximum extended video ad duration
+	MinBitrate    int       `json:"minbitrate,omitempty"`  // Minimum bit rate in Kbps
+	MaxBitrate    int       `json:"maxbitrate,omitempty"`  // Maximum bit rate in Kbps
+	Delivery      []int     `json:"delivery,omitempty"`    // List of supported delivery methods
+	CompanionAd   []Banner  `json:"companionad,omitempty"`
+	CompanionType []int     `json:"companiontype,omitempty"`
+	MaxSequence   int       `json:"maxseq,omitempty"`        // The maximumnumber of ads that canbe played in an ad pod.
+	Feed          int       `json:"companiontype,omitempty"` // Type of audio feed.
```

Comment:
It would be nice to see a few test with fixtures covering the new additions, something like https://github.com/bsm/openrtb/blob/master/testdata/breq.video.json. Not sure if OpenRTB 2.4 has something included in the doc